staging/src/k8s.io/cli-runtime/pkg/resource/query_param_verifier_test.go

```diff
@@ -69,6 +69,16 @@ func TestSupportsQueryParam(t *testing.T) {
 			supports:   false,
 			queryParam: QueryParamFieldValidation,
 		},
+		{
+			gvk: schema.GroupVersionKind{
+				Group:   "",
+				Version: "v1",
+				Kind:    "List",
+			},
+			success:    false,
```

Comment:
@seans3 @apelisse `List` type is not defined in swagger.json  https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/kubectl/testdata/openapi/swagger.json. Is `success: false` correct or should be defined in swagger.json?

Reply:
`success: false` is the correct value here; I don't think it needs to be defined in the fake swagger.json.

Reply:
Thanks @seans3 :)

Reply:
Yeah, it definitely should not be included in the openapi. But the question remains, why do we call this function with the list?